wiktor_hardware_interface/Robotiq3FingerActuatorCommand finger_a_command
wiktor_hardware_interface/Robotiq3FingerActuatorCommand finger_b_command
wiktor_hardware_interface/Robotiq3FingerActuatorCommand finger_c_command
wiktor_hardware_interface/Robotiq3FingerActuatorCommand scissor_command
std_msgs/Header header
